source/docs/advanced/performance/cache-control-and-cdn.md

```diff
@@ -0,0 +1,76 @@
+---
+id: cache-control-and-cdn
+title: Cache Control and CDN
+---
+
+<blockquote class="feature--new">
+  _This feature has been added in version `2.6.0`_
+</blockquote>
+
+To enable CDN caching you need to first set up proper [cache headers](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Cache-Control) for your site.
+
+In Front-Commerce we have implemented a mechanism where you can easily set up the `cache-control` headers for different parts of your site. You can also provide a custom implementation of the `cache-control` if needed (e.g. products with old last modified have longer caching duration).
+
+# Category/Product/CMS Pages
```

Comment:
We must document it in an agnostic way I think.\
While the default theme only contains these fragments, the documentation must directly explain how it works.

These fragments will be in the skeleton when the project will start, so developers will have faced it.\
We could however mention something more generic like:
> The default theme already contains specific fragments that you can override to customize Cache-Control settings for the most common pages. Please look for the `*CacheControlFragment.gql` files across the theme.

A section in the migration guide would be good though to highlight these new cache control fragments that they could override (Category / Product / CMS Page).

Reply:
I have a part related to cache implementation for a custom module